nisystemlink/clients/dataframe/_data_frame_client.py

```diff
@@ -21,7 +22,7 @@ def __init__(self, configuration: Optional[core.HttpConfiguration] = None):
                 is used.
 
         Raises:
-            ApiException: if unable to communicate with the Data Frame service.
+            ApiException: if unable to communicate with the DataFrame Service.
```

Comment:
Sneaking in these doc changes, after a conversation with Mark and Paul:
It's the DataFrame (one word) Service (capitalized), but the tables are data tables, not DataFrame tables.